tests/visco_plastic_transition_pressures_simple.prm

```diff
@@ -0,0 +1,87 @@
+# Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, single Stokes
+set Max nonlinear iterations               = 1
+set Surface pressure                       = 3.e9
```

Comment:
```suggestion
set Surface pressure                       = 1.55e9
```
To have the density change occur in the middle of the 100 km tall box, you want the pressure at 50 km to be 2.9e9, so subtract 50,000*10*2700 = 1.35e9  from 2.9e9 gives 1.55e9.

Reply:
whoops... that should say 50,000 x 10 x 2700.
if you look in the gnu-plot solution file you see that all your densities are 3400. this is because the
surface pressure is such that you are already deeper than the pressure change.  Also, see below, you need the gravity to be non-zero, otherwise pressure is constant